app/src/main/java/com/cornellappdev/uplift/models/FacilityInfo.kt

```diff
@@ -0,0 +1,26 @@
+package com.cornellappdev.uplift.models
+
+data class GymnasiumInfo(
+    val hours : List<TimeInterval>,
+    val courts : List<CourtInfo>
+)
+
+data class CourtInfo(
```

Comment:
add some comments on each of the info instead of having it in Gym.kt. I thought it was missing stuff until I read the Gym.kt comments

Reply:
but if each courtInfo is suppose to have the info for each day, you should have the name of the court out of this data class or else you would be repeating it for 7 time

Reply:
Basically, how this is structured is that "Gymnasiums" each day have hours and can have one or more courts. Each court also has its own hours and name (e.g. Basketball court, Volleyball court). The reason I decided to structure CourtInfo to have its own name each time is that I am unsure if the courts available on a specific day at a Gymnasium differ day to day. This structure gives that flexibility (although you may be right in that I won't need to account for that).

Reply:
got it then this is great